### DISCUSSION TITLE
tab+enter when typing in a code language

### DISCUSSION THREAD

**Discussion by extoplasm** [monkeytypegame/monkeytype#5334] [Questions] [answered]
When typing in any code language, tab+enter to next test as a shortcut does not work.

I understand that it's like a tab in code, but this behaviour is not present in any other languages and code has autotab anyways.

Should this be changed? It's really bugging me out that I can't quickly next test.

**Comment by Miodec** [comment, marked as answer]
You could, for example, try setting quick restart to esc in the settings

**Reply by extoplasm** [reply]
yea but esc is console, but i get what you mean

i didn’t know about that before, thanks!
